app/controllers/application_controller.rb

```diff
@@ -8,4 +10,13 @@ def authenticate_token
              status: :unauthorized
     end
   end
+
+  def current_user
+    @current_user
+  end
+
+  rescue_from CanCan::AccessDenied do |exception|
```

Comment:
Unused block argument - `exception`. You can omit the argument if you don't care about it.